src/darwin/Framework/CHIP/Matter.h

```diff
@@ -17,14 +17,6 @@
 
 #import <Foundation/Foundation.h>
 
-#ifndef MTR_NEWLY_DEPRECATED
-#define MTR_NEWLY_DEPRECATED(message)
-#endif
-
-#ifndef MTR_NEWLY_AVAILABLE
-#define MTR_NEWLY_AVAILABLE
-#endif
```

Comment:
Why is this being removed?

Reply:
Moved to `MTRDefines.h`, leaving `Matter.h` as a pure umbrella header.